### DISCUSSION TITLE
Add NetBeans to Localization wiki page

### DISCUSSION THREAD

**Discussion by henriquemeira** [dbeaver/dbeaver#16661] [Ideas]
Here how to edit bundle.properties file in NetBeans IDE.

Just load the maven project, under Files tab, select any bundle file, click with right mouse button and select Open.

See a screenshot sample attached.
![Screenshot_20210705_142053](https://user-images.githubusercontent.com/207619/124503711-802cfa00-dd9c-11eb-8ef5-4c30218bb63e.png)

**Comment by LonwoLonwo** [comment]
Is it working without any specific extensions in NetBeans?

**Reply by henriquemeira** [reply]
Yes, it's a native Netbeans features.
